core/src/main/java/io/github/mabartos/engine/DefaultRiskEngine.java

```diff
@@ -115,6 +121,13 @@ public void evaluateRisk(RiskEvaluator.EvaluationPhase phase, RealmModel realm,
         logger.debug("--------------------------------------------------");
     }
 
+    @Override
+    public boolean isRiskBasedAuthnEnabled() {
+        return Optional.ofNullable(this.realm.getAttribute(RISK_BASED_AUTHN_ENABLED_CONFIG))
+                .map(Boolean::parseBoolean)
+                .orElse(true); // Default to enabled if not configured
```

Comment:
+1 It's ok to have the default true. PS: If this code lies in the official Keycloak repo, the feature will be used.